src/app/api/services/edit-page/edit-page.service.spec.ts

```diff
@@ -35,7 +39,13 @@ describe('EditPageService', () => {
     beforeEach(() => {
         lastConnection = [];
 
-        this.injector = DOTTestBed.resolveAndCreate([EditPageService]);
+        this.injector = DOTTestBed.configureTestingModule({
+            providers: [EditPageService, DotRouterService, {
+                provide: LoginService,
+                useClass: LoginServiceMock
+            }],
+            imports: [RouterTestingModule]
+        });
```

Comment:
Really we are not checking if the interceptor is in used to do the request in this service, the implementation could use core-web-service instead and the test keep passing

Reply:
Great catch! done.